src/Simplex/Messaging/Notifications/Server/Store/Postgres.hs

```diff
@@ -638,22 +642,27 @@ importNtfSTMStore NtfPostgresStore {dbStore = s} stmStore = do
           case M.lookup (tokenKey tkn) deviceTokens of
             Just ts
               | length ts < 2 -> pure True
+              | ntfTknId `S.member` skipTokens -> False <$ putStrLn ("Skipped token " <> enc ntfTknId <> " from --skip-tokens")
               | otherwise ->
                   readTVarIO tknStatus >>= \case
                     NTConfirmed -> do
                       hasSubs <- maybe (pure False) (\v -> not . S.null <$> readTVarIO v) $ M.lookup ntfTknId tokenSubs
                       if hasSubs
                         then pure True
                         else do
-                          anyActive <- anyM $ map (\NtfTknData {tknStatus = tknStatus'} -> (NTActive ==) <$> readTVarIO tknStatus') ts
-                          if anyActive
+                          anyBetterToken <- anyM $ map (\NtfTknData {tknStatus = tknStatus'} -> activeOrInvalid <$> readTVarIO tknStatus') ts
```

Comment:
Invalid would be considered better than confirmed?

Reply:
yes